index.css

```diff
@@ -462,29 +535,370 @@ label {
   line-height: 28px;
 }
 
-.btn {
-  width: 140px;
-  height: 48px;
-  padding: 12px 16px;
-  border-radius: 8px;
-  color: #6070ff;
-  background-color: #fff;
-}
-
 .btn:hover {
   padding: 12px 16px 12px 16px;
   border: 1px solid #fff;
   color: #fff;
   background-color: #6070ff;
 }
 
-@media only screen and (min-width: 425px) {
+@media only screen and (min-width: 768px) {
+  .bg1 {
+    background-color: #f4f5f7;
+  }
+
+  .bg2 {
```

Comment:
- I'd like to kindly suggest that you make your portfolio responsive. On screensize 768px,and 1,440px there are some empty spaces on the right size. Can we ensure that all the contents of the portfolio fits perfectly on any screen size.

Here are some examples of what was displayed on the browser 👇🏽 

Here is what I can see compared to what's on Figma 👇🏽 
| Excess space on screensize 768px  |
| :---:   | 
|<img width="782" alt="Screenshot 2023-01-21 at 07 36 33" src="https://user-images.githubusercontent.com/99246359/213869813-11d17eff-ff54-4240-adc6-c05969e74bbb.png">  |

|Excess space on screensize 768px |
| :---:   | 
|<img width="774" alt="Screenshot 2023-01-21 at 07 36 51" src="https://user-images.githubusercontent.com/99246359/213869827-40aa5756-1ce3-4a0b-b014-fc809609fbc4.png">  |

|Excess space on screensize 1,440px |
| :---:   | 
|<img width="1026" alt="Screenshot 2023-01-21 at 07 37 36" src="https://user-images.githubusercontent.com/99246359/213869835-176ccf4d-2d6e-431e-b389-c497bbe49fa9.png">  |


⚠️ Kindly take note of these sections and ensure that it's responsive on different screen sizes 🌸

Reply:
resolve the desktop view accordingly. made the changes in index.css file.